Documents/Management/WeeklyReports/mainWeeklyReport.md

```diff
@@ -0,0 +1,76 @@
+
+# mainWeeklyReport
+
+<details>
+<summary>Table of contents</summary>
+
+- [Week 1](#weeklyreportfirstweek)
+
+# WeeklyReportFirstWeek
+
+[Week 1](https://github.com/algosup/2023-2024-project-4-sportshield-team-2/Documents/Management/weeklyReportFirstWeek.md)
+
+</details>
+
+## Week 1 (11-15 March 2024)
+
+The first week was dedicated to initiating the project and planning. The team discuss about project and the tasks to be done. The functional specifications document was started. The team also began to reverse-engineer the existing software to understand how it works.
+
+## Topics of the week
+
+1. Read and analyse the call for tender.
+2. Meeting with the group to discuss the project.
+3. Begin to write the functional specifications document.
+4. Reverse engineering of the existing software.
+5. Gather questions we have about the project
+6. Ask the client for a meeting to clarify some points.
+
+## Achievements
+
+1. Kick-off presentation by the client.
+    - The client presented the project to the team.
+    - Analyse the call for tender
+
+2. Documents
+    - Functional specifications are well advanced.
+    - Project charter is done.
+    - Test plan is started.
+    - User manual is started.
+    - The RACI matrix is created.
+
+3. Hardware components
+    - The team has received all the hardware components.
+    - The team has tested the hardware components.
+    - The team has started to work on the hardware components.
+
+4. Reverse engineering
+    - The team has started to reverse engineer the existing software.
+
+## Challenges
+
+1. Hardware part
+    - The team had some trouble with the hardware, as the components were not working as expected.
+    - The team decided to take other components that were working properly.
+2. Reverse engineering
+    - The team had some trouble with the reverse engineering of the existing software mainly because of the hardware components.
+3. Working for a real company
+    - For some of the team members, it was their first time working for a real company, so it's different for them but they are adapting.
+
+## Next week's plan
+
+1. Documents
+    - Finish the functional specifications document on Monday.
+    - Add the KPIs to the management documents.
+    - Begin the technical specifications document.
+    - Continue the test plan.
+    - Continue the user manual.
+    - Begin the statement of work.
+
+2. Hardware components
+    - Solve the problems with the hardware components.
```

Comment:
Specify that the problems are just with the SIM card, rather than the hardware as a whole? Given that the only problem we have at the moment is with our SIM card.